controller/src/main/java/io/pravega/controller/store/stream/ZkInt96Counter.java

```diff
@@ -119,26 +119,28 @@ public ZkInt96Counter(ZKStoreHelper storeHelper) {
 
     @VisibleForTesting
     CompletableFuture<Void> getRefreshFuture() {
-        return storeHelper.createZNodeIfNotExist(COUNTER_PATH, Int96.ZERO.toBytes())
-                          .thenCompose(v -> storeHelper.getData(COUNTER_PATH, Int96::fromBytes)
-                                                       .thenCompose(data -> {
-                                                           Int96 previous = data.getObject();
-                                                           Int96 nextLimit = previous.add(COUNTER_RANGE);
-                                                           return storeHelper.setData(COUNTER_PATH, nextLimit.toBytes(), data.getVersion())
-                                                                             .thenAccept(x -> {
-                                                                                 // Received new range, we should reset the counter and limit under the lock
-                                                                                 // and then reset refreshfutureref to null
-                                                                                 synchronized (lock) {
-                                                                                     // Note: counter is set to previous range's highest value. Always get the
-                                                                                     // next counter by calling counter.incrementAndGet otherwise there will
-                                                                                     // be a collision with counter used by someone else.
-                                                                                     counter.set(previous.getMsb(), previous.getLsb());
-                                                                                     limit.set(nextLimit.getMsb(), nextLimit.getLsb());
-                                                                                     refreshFutureRef = null;
-                                                                                     log.info("Refreshed counter range. Current counter is {}. Current limit is {}", counter.get(), limit.get());
-                                                                                 }
-                                                                             });
-                                                       }));
+        return storeHelper
```

Comment:
What is the change here? Perhaps a formatting change that needs to be reverse...

Reply:
yes, its only formatting. have kept such formatting changes here too because there were many places in our store code where the 120 char width was not adhered to (esp here) making the code difficult to read thru.